render.js

```diff
@@ -0,0 +1,24 @@
+const Renderer = function () {
+
+    const addPost = function (post) {
+        $('#posts').append(`<div class="post" data-id="${post.id}"></div`)
+        $(`[data-id="${post.id}"]`).append(`<button class="delete">Delete Post</button>`)
+        $(`[data-id="${post.id}"]`).append(`<p class="post-text">${post.text}</p>`)
+        for (comment of post.comments) {
```

Comment:
When declaring a new variable you should declare it with  _let_, otherwise, the variable will be defined as "_var_"